test/e2e/secret_file_permission_test.go

```diff
@@ -98,7 +98,7 @@ var _ = Describe("When user provides file permission for secrets", func() {
 	})
 
 	It("should mount secret file with given permission", func() {
-		if !config.IsKindCluster {
+		if !config.IsKindCluster || config.IsConformanceTest {
```

Comment:
Instead of this, let's use `ginkgo.skip=${GINKGO_SKIP}` and skip the test for conformance test pipeline. That way for newer releases the env variable can be unset and we don't need to update it here.